server/database/queries/getAvailableRooms.js

```diff
@@ -0,0 +1,8 @@
+const dbconniction = require('../db_connection')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
```

Comment:
This is suspicious, what is this for? Environment variables shouldn't be defined/modified in the code unless really necessary (can't think of a scenario where it actually would). For the app's state (especially mutable state) use regular variables, for constants like this one an environment variable is appropriate, however it should be [defined ](https://medium.com/@trekinbami/using-environment-variables-in-react-6b0a99d83cf5)in an `.env` file.
However this looks more like it got here by mistake so pls remove it.

Reply:
Actually it was a solution for certificate error we had, I found it as a solution and yes somehow It worked!!

Reply:
hmm, where is the certificate coming from (where are you trying to connect to)? [Ideally](https://stackoverflow.com/questions/35633829/node-js-error-process-env-node-tls-reject-unauthorized-what-does-this-mean) this should be fixed by fixing the certificate, however for your purposes it might be ok as a quickfix.
In any case, pls move it to the .env file 🙂